projects/applicationinsights-angularplugin-js/src/test.ts

```diff
@@ -4,22 +4,13 @@ import "zone.js";
 import "zone.js/testing";
 import { getTestBed } from "@angular/core/testing";
 import { BrowserDynamicTestingModule, platformBrowserDynamicTesting} from "@angular/platform-browser-dynamic/testing";
-
-declare const require: {
-  context(path: string, deep?: boolean, filter?: RegExp): {
-    <T>(id: string): T;
-    keys(): string[];
-  };
-};
+import "./lib/applicationininsights-angularplugin-mulple-instace.spec";
```

Comment:
This looks wrong.

it should be 
`import { xxxx } from "./lib/application...`;

Or just not here and when the test run they should pick up all `*.spec.ts` files